cmake/schemes/url_sha1_cmake.cmake.in

```diff
@@ -150,6 +150,8 @@ foreach(configuration @HUNTER_PACKAGE_CONFIGURATION_TYPES@)
       "${build_type_opts}"
       "-DCMAKE_INSTALL_PREFIX=@HUNTER_PACKAGE_INSTALL_PREFIX@"
       "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}"
+      "-DHUNTER_DOWNLOAD_SERVER=@HUNTER_DOWNLOAD_SERVER@"
```

Comment:
Move them to HUNTER_DOWNLOAD_TOOLCHAIN.

Reply:
I don't understand what I should move where. Do you mean I should rename `HUNTER_DOWNLOAD_SERVER` to `HUNTER_DOWNLOAD_TOOLCHAIN`?

Reply:
See https://github.com/ruslo/hunter/blob/93dc8ab9cd221238a71efbdae2279ae3233cd54d/cmake/modules/hunter_download.cmake#L393-L432

Reply:
can I append the two variables at once to the file, or should I use two `file(APPEND)` calls?

Reply:
I think you can use one `file(APPEND)` command, but to keep things consistent it make sense to use two calls to fit current style. This part should be refactored anyway, so I will not be too picky about it here.